client/utils/auth.py

```diff
@@ -33,34 +30,35 @@
 CONFIG_DIRECTORY = os.path.join(os.path.expanduser('~'), '.config', 'ok')
 
 REFRESH_FILE = os.path.join(CONFIG_DIRECTORY, "auth_refresh")
-REDIRECT_HOST = "localhost"
+REDIRECT_HOST = "127.0.0.1"
 TIMEOUT = 10
 
 SERVER = 'https://ok.cs61a.org'
+DEFAULT_PORT = 6165
```

Comment:
I guess this should be moved together with `REDIRECT_HOST`, might as well call it `REDIRECT_PORT` too.